File: test/new_lang/test6/TestProgram.pas
program TestProgram;
uses SplashKit;

procedure Main();
var
  v: Vector2D;
begin
  v := GetVector();
  PrintVector(v);
  WriteLn('Matches: ', v.x, ',', v.y, ' and ', v.MultiName);
end;

begin
  Main();
end.

File: test/new_lang/test6/TestProgram.cs
using static SplashKitSDK.SplashKit;
using static System.Console;
using System.Collections.Generic;

public class Program
{
  public static void Main()
  {
    Vector2D v;

    v = GetVector();
    PrintVector(v);
    WriteLine("Matches: {0},{1} and {2} is mapped {3}", v.x, v.y, v.multiName, v.mapped);
  }
}
